content/millennium/r4/base/entities/location.md

```diff
@@ -53,11 +53,18 @@ Search for Locations that meet supplied query parameters:
 
 ### Parameters
 
- Name            | Required?               | Type       | Description
------------------|-------------------------|------------|-------------------------------------------------------
- `_id`           | This or `-physicalType` | [`token`]  | The logical resource id associated with the resource.
- `-physicalType` | This or `_id`           | [`token`]  | The location’s physical type. Example: `http://terminology.hl7.org/CodeSystem/location-physical-type|ro`
- [`_count`]      | no                      | [`number`] | The maximum number of results to return. Defaults to `100`.
+ Name                 | Required?               | Type         | Description
+----------------------|-------------------------|--------------|-------------------------------------------------------
+ `_id`                | This or `-physicalType` | [`token`]    | The logical resource id associated with the resource.
+ `-physicalType`      | This or `_id`           | [`token`]    | The location’s physical type. Example: `http://terminology.hl7.org/CodeSystem/location-physical-type|ro`
+ [`_count`]           | no                      | [`number`]   | The maximum number of results to return. Defaults to `100`.
+ `address`            | no                      | [`string`]   | A (part of the) address of the location.
+ `address-city`       | no                      | [`string`]   | A city specified in an address
+ `address-state`      | no                      | [`string`]   | A state specified in an address
+ `address-postalCode` | no                      | [`string`]   | A postal code specified in an address
+ `name`               | no                      | [`string`]   | A portion of the location's name or alias
+ `organization`       | no                      | [`reference`]| Searches for locations that are managed by the provided organization
```

Comment:
Can you include the examples for name and organization serach

Reply:
added 5d54dc306a2722c759d56ff65ad300ddcc83078b